modules/multi-arch-uninstalling-using-cli.adoc

```diff
@@ -0,0 +1,52 @@
+//Module included in the following assemblies
+//
+//post_installation_configuration/multiarch-tuning-operator.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="multi-architecture-uninstalling-using-cli_{context}"]
+
+= Uninstalling Multiarch Tuning Operator by using the CLI
+
+You can uninstall the Multiarch Tuning Operator by using the {oc-first}.
+
+.Prerequisites
+
+* You have access to cluster with `cluster-admin` permissions.
+* You have installed the {oc-first}.
+* You have deleted the `PodPlacementConfig` custom resource (CR).
+
+.Procedure
+
+. Log in to the {oc-first}.
+
+. Delete the `OperatorGroup` object by running the following command:
++
+[source,terminal]
+----
+$ oc delete operatorgroup <file_name> -n <namespace> <1>
```

Comment:
Is this the way other operators are uninstalled?

Reply:
@aleskandro 
I am not sure about "Uninstalling using the CLI". It seems this proc is incomplete/incorrect.
Could you share the right way to uninstall this Operator using the CLI?
Thanks

Reply:
should we just link the users to [this](https://docs.openshift.com/container-platform/4.15/operators/admin/olm-deleting-operators-from-cluster.html)

Reply:
I think it will be best to document this info within the same chapter instead of redirecting users to generic info.

Reply:
ack. That link has all the info on how to uninstall (you are mostly missing the deletion of the CSV).

Reply:
Updated the Uninstallation proc.